[2024-07-24T15:07:00.267+0900] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-24T15:07:00.275+0900] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: import_db.Check scheduled__2024-07-16T00:00:00+00:00 [queued]>
[2024-07-24T15:07:00.277+0900] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: import_db.Check scheduled__2024-07-16T00:00:00+00:00 [queued]>
[2024-07-24T15:07:00.277+0900] {taskinstance.py:2306} INFO - Starting attempt 3 of 3
[2024-07-24T15:07:00.281+0900] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): Check> on 2024-07-16 00:00:00+00:00
[2024-07-24T15:07:00.283+0900] {standard_task_runner.py:64} INFO - Started process 94385 to run task
[2024-07-24T15:07:00.286+0900] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'import_db', 'Check', 'scheduled__2024-07-16T00:00:00+00:00', '--job-id', '249', '--raw', '--subdir', 'DAGS_FOLDER/import_db.py', '--cfg-path', '/var/folders/qv/0knwm5h50ml0skpppwpbks_r0000gp/T/tmpxdqkpj4q']
[2024-07-24T15:07:00.287+0900] {standard_task_runner.py:91} INFO - Job 249: Subtask Check
[2024-07-24T15:07:00.301+0900] {task_command.py:426} INFO - Running <TaskInstance: import_db.Check scheduled__2024-07-16T00:00:00+00:00 [running]> on host jeongui-MacBookPro.local
[2024-07-24T15:07:00.323+0900] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='import_db' AIRFLOW_CTX_TASK_ID='Check' AIRFLOW_CTX_EXECUTION_DATE='2024-07-16T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-16T00:00:00+00:00'
[2024-07-24T15:07:00.324+0900] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-24T15:07:00.329+0900] {subprocess.py:63} INFO - Tmp dir root location: /var/folders/qv/0knwm5h50ml0skpppwpbks_r0000gp/T
[2024-07-24T15:07:00.330+0900] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'bash /Users/DE32/airflow/sh/simple_done_check.sh ~/data/done/20240716/_DONE']
[2024-07-24T15:07:00.333+0900] {subprocess.py:86} INFO - Output:
[2024-07-24T15:07:00.336+0900] {subprocess.py:93} INFO - _DONE Path = /Users/DE32/data/done/20240716/_DONE
[2024-07-24T15:07:00.336+0900] {subprocess.py:93} INFO - /Users/DE32/airflow/sh/simple_done_check.sh: line 7: figlet: command not found
[2024-07-24T15:07:00.336+0900] {subprocess.py:97} INFO - Command exited with return code 0
[2024-07-24T15:07:00.337+0900] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-24T15:07:00.343+0900] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=import_db, task_id=Check, run_id=scheduled__2024-07-16T00:00:00+00:00, execution_date=20240716T000000, start_date=20240724T060700, end_date=20240724T060700
[2024-07-24T15:07:00.348+0900] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-07-24T15:07:00.356+0900] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-07-24T15:07:00.357+0900] {local_task_job_runner.py:222} INFO - ::endgroup::
